.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] kindly consider including the test.md  file  so that it wont be pushed on your git hub and doing so will help you to full-fill the below requirement
> A test.md file with the text "This file should be ignored by git".
> Make sure that the last file is actually ignored by git and is not present in your GitHub repository.